app/src/ui/changes/commit-message-avatar.tsx

```diff
@@ -150,7 +176,7 @@ export class CommitMessageAvatar extends React.Component<
           <div>
             The email in your global Git config (
             <span className="git-email">{this.props.email}</span>) doesn't match
-            your GitHub{accountTypeSuffix} account.{' '}
+            your GitHub{accountTypeSuffix} account{userName}.{' '}
```

Comment:
Since the avatar no longer has a tooltip when there is a warning and therefore the user cannot see the GitHub user they are logged in as, I thought adding the username makes sense in this context.  (Tho open to opinions since technically users should only have one GitHub account.. but I think the reason this warning is here is because they are maybe using a personal and a work account :/) 

![CleanShot 2023-04-06 at 07 09 55](https://user-images.githubusercontent.com/75402236/230360479-debf2667-8d3f-4045-8acb-d41e129180a6.png)